packages/frontend/components/layout/MobileMenuToggle/MobileMenuToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import * as Styled from "./MobileMenuToggle.styles";
+import { IconFactory } from "components/factories";
+type IconProps = React.ComponentProps<typeof IconFactory>;
+
+export default function MobileMenuToggle({ onToggle, label, icon }: Props) {
+  return (
+    <Styled.Toggle onClick={onToggle}>
```

Comment:
Unless I missed it, it looks like we're missing the a11y attribute `aria-controls` for this toggle https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls